CHANGELOG.unreleased.md

```diff
@@ -42,6 +42,21 @@ Examples:
 
 -->
 
+- JavaScript: Correctly handle comments in empty arrow function expressions ([#6086] by [@evilebottnawi])
+
+   <!-- prettier-ignore -->
+
+  ```js
+  // Input
+  const fn = () => /*event, data*/ doSomething(anything);
+
+  // Output (Prettier stable)
+  const fn = () => /*event, data*/ doSomething(anything);
```

Comment:
Should these two examples have the comment inside the `()`?

Reply:
Typo, fixed :+1: